pom.xml

```diff
@@ -30,8 +30,8 @@
     </licenses>
 
     <scm>
-        <connection>scm:git:https://github.com/openhab/openhab.git</connection>
-        <developerConnection>scm:git:https://github.com/openhab/openhab.git</developerConnection>
+        <connection>scm:git:${scm.gitBaseUrl}/openhab.git</connection>
```

Comment:
Could you change the repo name to openhab1-addons.git (in both lines), which is the current one? Thanks!

Reply:
done!